ios/Podfile.lock

```diff
@@ -1980,9 +1980,9 @@ SPEC CHECKSUMS:
   SDWebImageWebPCoder: af09429398d99d524cae2fe00f6f0f6e491ed102
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
   Turf: 13d1a92d969ca0311bbc26e8356cca178ce95da2
-  VisionCamera: 7d13aae043ffb38b224a0f725d1e23ca9c190fe7
+  VisionCamera: fda554d8751e395effcc87749f8b7c198c1031be
   Yoga: e64aa65de36c0832d04e8c7bd614396c77a80047
 
 PODFILE CHECKSUM: 0ccbb4f2406893c6e9f266dc1e7470dcd72885d2
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.14.3
```

Comment:
Let's not bump pod version here. It's out of scope

Reply:
Yeah, I was aware of that, it is fixed already